test_setup/init.sls

```diff
@@ -1,82 +1,8 @@
 {# Import global parameters that source from grains and pillars #}
 {% import 'params.jinja' as params %}
 
-{# This is a current workaround for fips issue documented here: https://github.com/saltstack/salt/issues/40890 #}
-{% set fips_enabled = salt['cmd.run']('cat /proc/sys/crypto/fips_enabled') %}
-
-{% if fips_enabled == '1' %}
-fips_is_enabled:
-  cmd.run:
-    - names:
-      - echo testing_fips
-{% endif %}
-
-{% if params.on_smartos %}
-{% set update_path = salt['environ.get']('PATH', '') + ':/opt/salt/bin/' %}
-add_saltkey_path:
-   environ.setenv:
-     - name: PATH
-     - value: {{ update_path }}
-     - update_minion: True
-
-disable_services:
-  cmd.run:
-    - names:
-      - svcadm disable salt:minion
-      - svcadm disable salt:master
-{% else %}
-disable_services:
-  service.dead:
-    - names:
-      - salt-master
-      - salt-minion
-{% if params.os == 'Debian' %}
-    - init_delay: 10
-{% endif %}
-    - require_in:
-      - file: remove_pki
-      - file: clear_minion_id
-      - file: minion_config
+include:
+  - test_setup.master_minion
+{% if params.os in ('CentOS', 'Redhat', 'Amazon', 'Debian', 'Ubuntu') %}
```

Comment:
ping @dmurphy18 these are the OSs we will be testing salt-api functionality. Based on what you provide during the building of the packages should I test more or is this sufficient?